pages/api/event/event.interface.ts

```diff
@@ -18,6 +18,7 @@ export interface EventData {
   photos: string[]
   tags: string[]
   capacity:number
+  attendeeCount:number
   category: string
   speakers: SpeakerData[]
   attendees:AttendeeData[]
```

Comment:
@Akalanka47000 are u planning to update the Event record when new users RSVP to the event?

Reply:
yes

Reply:
Don't merge this PR though until the latest PR to the backend is confirmed as okay, this is in sync with that

Reply:
Okay then